repo/meta/version.json

```diff
@@ -1,4 +1,4 @@
 {
-  "version": "0.1.0-alpha"
+  "version": "1.0.0-rc1"
```

Comment:
It seems this pull request is for the master branch. Maybe we should have created it for the version-1.x branch? These changes seem to already be in the version-1.x branch, but not in master. Therefore they show up here in this pull request display.